### DISCUSSION TITLE
Make 'Save Tab Groups for Tab Session Manager' work with Edge

### DISCUSSION THREAD

**Discussion by DrewNeilson** [sienori/Tab-Session-Manager#1043] [Ideas]
Save Tab Groups for Tab Session Manager isn't working for me in Microsoft Edge. I use the current general public release version of Edge on the current general public release version of Windows 10, as of today's date, 8-19-22. I installed Tab Session Manager from Microsoft's Edge browser add-ons page/store/repository, and I installed Save Tab Groups for Tab Session Manager from Chrome's web store. Tab Session Manager works, but Save Tab Groups for Tab Session Manager does not.
EDIT: To clarify: Tab Session Manager saves tabs, but does not preserve the groups that they are in, and so when a previously saved session is restored, tabs open, but not in the groups that they were in when the session was saved.

**Comment by blinkertoon** [comment]
Hi. Wondered about this extra option but still don't understand the difference between saving tabs (tree style by window) and in groups - unless you mean the new Edge group feature? (Sorry if missing the obvious ;)). Thx

**Reply by xtchsbrlx** [reply]
If it's not about supporting the Group Tab feature in Edge, then I would like to also suggest that haha. I've been looking for a tab manager that can do that, since I sometimes move tabs/sessions between stations and it's annoying to have a huge mess of tabs that I have to manually group again.

**Comment by thomas-neuburger** [comment]
I have a similar problem with Save Tab Groups and Brave. When I browse the sessions I see nothing about groups. 

Note: I haven't tried to restore a session yet, or a multi-tab window. But still, something about my groups should appear in the data, I would think. For example, what if I wanted to restore just a group?

Thanks for attending to this.

Thomas
